app/worker/http.go

```diff
@@ -39,33 +40,42 @@ var makeHTTPRequest = func(req *http.Request, timeout time.Duration) (*http.Resp
 	return res, nil
 }
 
-func processHTTPTask(httpTask *model.HTTPTask, job *model.Job,
-	workflow *model.Workflow, l *log.Logger) (*model.TaskResult, error) {
-	uri := processJobString(httpTask.URI, workflow, job)
+func processHTTPTask(
+	httpTask *model.HTTPTask,
+	ps *processor.JobStringProcessor,
+	l *log.Logger,
+) (*model.TaskResult, error) {
+	options := &processor.Options{Encoding: processor.URL}
+	uri := ps.ProcessJobString(httpTask.URI, options)
+	l.Infof("processHTTPTask: starting with: method=%s uri=%s (options=%+v)",
+		httpTask.Method,
+		uri,
+		options,
+	)
```

Comment:
I don't think logging option is valuable, as it will always be Encoding: URL.
By the way, I don't get why we need this log here.

Reply:
I can remove it, it was moved from one level up (see the previous comment). do you want me to remove it?